metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/ELASTICSEARCH/2.3.3/configuration/elastic-site.xml

```diff
@@ -189,7 +189,12 @@
     </property>
     <property>
         <name>network_host</name>
-        <value>"_lo:ipv4_","_eth0:ipv4_"</value>
-        <description>Network interface(s) ES will bind to within each node. Confirm names via ifconfig.  Should have entries for primary external and loopback interfaces, with :ipv4 annotation and quote marks around each entry: "_lo:ipv4_","_eth0:ipv4_"</description>
+        <value>[ _local_ ]</value>
+        <description>Network interface(s) ES will bind to within each node. Use [ _local_ ] for single-node install; change to [ _local_, _site_ ] (including square brackets) for typical cluster install. See https://www.elastic.co/guide/en/elasticsearch/reference/2.3/modules-network.html for ES documentation.</description>
+    </property>
+    <property>
+        <name>network_publish_host</name>
```

Comment:
@nickwallen , excellent catch, thank you!  @JonZeolla , agree.  It appears the correct addition is:
```
<value-attributes>
    <empty-value-valid>true</empty-value-valid>
</value-attributes>
```
I will add this momentarily.  

@nickwallen , to attempt a quick test, one could stop ambari-agent and ambari-server, then edit the file /var/lib/ambari-server/resources/mpacks/metron-ambari.mpack-0.4.0.0/common-services/ELASTICSEARCH/2.3.3/configuration/elastic-site.xml to add those lines to the network_publish_host property.  But if the xml file has already been used to create the ambari db, it may be necessary to start over; sorry.